swagger-ui/src/main/resources/index.html

```diff
@@ -1,142 +0,0 @@
-<!--
```

Comment:
Isn't the swagger UI then also exposed to the end user ? If so, simply by its existence it already increase the attack surface considerably because since Swagger is a widely used framework this could make iPaaS vulnerable for automated attacks in case of an security breach.

So, imo because of maintenance and security we should keep it out in the _final version_. If we remove it now, we could be sure. Otherwise there is a chance we can forget to remove it.